script.js

```diff
@@ -22,16 +28,102 @@ ulCont.addEventListener('click', ((e) => {
+    imagesrc: 'imgs/poster 3.svg',
+    technologies: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+  },
+  {
+    name: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    imagesrc: 'imgs/poster 4.svg',
+    technologies: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+  },
+];
+
+const projectImgSrc = document.querySelectorAll('.tonic');
+const projectLanguagesHtml = document.querySelectorAll('.project-languages-html');
+const projectLanguagesRuby = document.querySelectorAll('.project-languages-ruby');
+const projectLanguagesCss = document.querySelectorAll('.project-languages-css');
+const projectLanguagesJs = document.querySelectorAll('.project-languages-js');
+const projectsDescription = document.querySelectorAll('.projects-description');
+
+for (let i = 0; i < projectLanguagesHtml.length; i++) {
+  projectImgSrc[i].src = storeData[i].imagesrc;
+  projectLanguagesHtml[i].innerHTML = storeData[i].technologies[0];
+  projectLanguagesRuby[i].innerHTML = storeData[i].technologies[1];
+  projectLanguagesCss[i].innerHTML = storeData[i].technologies[2];
+  projectLanguagesJs[i].innerHTML = storeData[i].technologies[3];
+  projectsDescription[i].innerHTML = storeData[i].description;
+}
+
+const popupsProjectImage = document.querySelector('.popup-img');
+const popupsProjectTitle = document.querySelector('.popups-proj-title');
+
+const openModal = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[0].imagesrc2;
+  popupsProjectTitle.innerHTML = storeData[0].name;
+};
+
+openModalBtn.addEventListener('click', openModal);
+
+const openModal2 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[1].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[1].name;
+  popupsProjectTitle.style.fontSize = '20px';
+};
+
+openModalBtn2.addEventListener('click', openModal2);
+
+const openModal3 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[2].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[2].name;
+  popupsProjectTitle.style.fontSize = '17px';
+};
+
+openModalBtn3.addEventListener('click', openModal3);
+
+const openModal4 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[3].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[3].name;
+};
```

Comment:
- [ ] When I clicked on the See Project button the modal window open on the top of the page it is supposed to show in the section where the user is, the same behavior on the mobile version. Also when I open your project or refresh it a modal window is in there, please fix it. 👇🏽 

> I would suggest we stick to the project requirement as much as possible, currently, the See project button is not functioning properly for the modal pop-up, only the button in the first card is responding to the modal pop-up when clicked on while others are not. I would suggest you create it dynamically and loop through the array.

https://user-images.githubusercontent.com/58642949/209444349-e4ca0c56-735f-49eb-953d-5d6cb5577855.mp4